### PR TITLE
Muenster: update scraper and geojson

### PR DESCRIPTION
**Pull request by hbruch**
This PR updates muenster scraper and geojson.

i.e., 
* it includes the table, as it's exclusion skippe the last parking
* in `muenster.get_lot_infos` adds minimally required lot info for yet unknown parking lots
* it adds missing parking lots to the geojson
* updates manually set capacity and has_live_data

### CODE REVIEW

**Comment by hbruch** [comment]
Sorry, not sure if I get it right: `has_live_capacity` is a property of LotInfo and should be contained in the geojson. It should only be true, if the capacity (Not availability!) is a life information which is updated live, e.g. because it is available anyway when in the source data. 

Is that documented already somewhere in the docs/code?

**Review by jklmnn — COMMENTED** [review]
Fine for me. Given that there is still an open discussion with I'll wait for an approval by @defgsus.

**Comment by hbruch** [comment]
With the latest switch to an official geojson datasource, capacity is updated live, also.
That should solve @defgsus' comment.

Nevertheless, I'd like to challenge the meaning of `has_live_capacity`, but that's for another discussion.

**Review by jklmnn — APPROVED** [review]

**Pull request merged**
